internal/restic/node_windows.go

```diff
@@ -114,7 +117,7 @@ func (s statT) mtim() syscall.Timespec {
 
 func (s statT) ctim() syscall.Timespec {
 	// Windows does not have the concept of a "change time" in the sense Unix uses it, so we're using the LastWriteTime here.
-	return syscall.NsecToTimespec(s.LastWriteTime.Nanoseconds())
+	return s.mtim()
```

Comment:
This fixes func statT.mtim is unused (U1000)go-staticcheck and removes code duplication